membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -270,7 +270,16 @@ class AccountController(commonActions: CommonActions, override val controllerCom
       stripeService = accountSummary.billToContact.country.map(RegionalStripeGateways.getGatewayForCountry).flatMap(tp.stripeServicesByPaymentGateway.get).getOrElse(tp.ukStripeService)
       alertText <- OptionEither.liftEitherOption(alertText(accountSummary, sub, getPaymentMethod))
       isAutoRenew = sub.autoRenew
-    } yield AccountDetails(contact.regNumber, accountSummary.billToContact.email, upToDatePaymentDetails, stripeService.publicKey, isAutoRenew, alertText).toJson).run.run.map {
+    } yield AccountDetails(
+      contact.regNumber,
+      accountSummary.billToContact.email,
+      upToDatePaymentDetails,
+      stripeService.publicKey,
+      accountHasMissedRecentPayments = false,
```

Comment:
This is the old (deprecated) endpoint. I was planning on checking if anything is still calling this and ripping it out if not...

Reply:
minor: I would rather pass all the params by name instead of some by name and some by position.